reply.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+from twbot import api
+import tweepy
+import time
+
+def reply():
```

Comment:
It would be nice if there was a docatring for this

Reply:
Done